portal/static/js/main.js

```diff
@@ -2112,8 +2112,12 @@ var assembleContent = {
 
             var studyIdField = $("#profileStudyId");
             var siteIdField = $("#profileSiteId");
-            var hasStudyId = studyIdField.length > 0 && studyIdField.is(":visible");
-            var hasSiteId = siteIdField.length > 0 && siteIdField.is(":visible");
+            /*
+             * siteId field is only present in Truenth
+             * studyId field is only present in Eproms
+             */
+            var hasStudyIdField = studyIdField.length > 0 && studyIdField.is(":visible");
```

Comment:
Would you be able to just check `studyId`, `siteId` and remove `hasStudyIdField`, `hasSiteIdField`? It doesn't seem like you the latter variables later.